### DISCUSSION TITLE
Tempo de Viagem

### DISCUSSION THREAD

**Discussion by JhanAragao** [YuriAlessandro/gerador-ficha-tormenta20#370] [Mapa Interativo de Arton]
Antes de mais nada, achei formidável a ideia do Mapa Interativo de Arton.

Porém, tenho uma pequeno sugestão à fazer, na opção de **_"Medir Distâncias (régua)"_**, poderia retirar os pontos de percursos, deixando apenas o ponto final da viagem, pois diversos pontos com distâncias, atrapalham um pouco, além de adicionar o botão/opção de ponto final.

Uma outra observação sobre **_"Medir Distâncias_**, poderia ser adicionado o tempo de viagem, como dias, facilitaria e muito a vida de qualquer mestre de Tormenta.

![Captura de tela 2024-04-03 1550545](https://github.com/YuriAlessandro/gerador-ficha-tormenta20/assets/97192540/921a9cfb-67fe-4936-8820-bf8c62eef313)

**Comment by YuriAlessandro** [comment]
Essa régua está bem bugada e fora de escala. Vou ter que refazer ela.

**Comment by YuriAlessandro** [comment]
Não demorou quase nada mas eu corrigi isso. Tem uma régua mais simples agora, mas ela mostra a distância corretamente em todos os pontos do mapa de acordo com a escala.
